src/content/reference/react-dom/components/form.md

```diff
@@ -0,0 +1,458 @@
+---
+title: "<form>"
+canary: true
+---
+
+<Canary>
+
+React's `form` Component is currently only available in React's canary and experimental channels but you can use the [standard HTML `form` tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form) in your React's stable channel. Learn more about [React's release channels here](/community/versioning-policy#all-release-channels).
+
+</Canary>
+
+
+<Intro>
+
+The [built-in browser `<form>` component](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form) lets you create interactive controls for submitting information.
+
+```js
+<form action={search}>
+    <input name="query" />
+    <button type="submit">Search</button>
+</form>
+```
+
+</Intro>
+
+<InlineToc />
+
+---
+
+## Reference {/*reference*/}
+
+### `<form>` {/*option*/}
+
+The [built-in browser `<form>` component](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form) lets you create interactive controls for submitting information.
+
+```js
+<form action={search}>
+    <input name="query" />
+    <button type="submit">Search</button>
+</form>
+```
+
+[See more examples below.](#usage)
+
+#### 🚧 Props {/*props*/}
+
+`<form>` supports all [common element props.](/reference/react-dom/components/common#props)
+
+Additionally, `<form>` supports these props:
+
+* [`action`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/option#disabled): A boolean. Specifies what to do when a form is submitted. The form data can be handled with client or server side code.
+* 🚧
+
+
+🚧 Forms can contain one or more of the following tags:
+* [`<input>`]
+* [`<textarea>`]
+* [`<button>`]
+    * [`formAction`] prop that overrides `<form>`'s `action` prop
+* [`<select>`]
+* [`<option>`]
+* [`<optgroup>`]
+* [`<fieldset>`]
+* [`<label>`]
+* [`<output>`]
+* 🚧
+
+#### 🚧 Caveats {/*caveats*/}
+
+* 🚧
+
+---
+
+## Usage {/*usage*/}
+
+### 🚧 Submitting forms to the server {/*progressively-enhanching-forms*/}
+
+Render a `<form>` with a input and submit button. Pass a function marked with `'use server';` to the `action` prop of form to run the function when the form is submitted.
```

Comment:
I don't feel we should have the semicolon  in 'use server' here when it's used inline in a sentence as it makes the sentence too hard to parse. Your code formatter will add the semicolon if you perchance don't know or remember to, so I don't think this will lead to frustration.

Reply:
Fixed!